### DISCUSSION TITLE
Computer vision model serving

### DISCUSSION THREAD

**Discussion by Udaykiran87** [iterative/dvc.org#4733] [Q&A]
Hello, Thanks for the wonderful work. Can you please help me to perform model serving using iterative.ai using mlem especially for computer vision application like object detection using yolov8, or any tensorflow model. Is there any tutorial to do hands dirty quickly?

**Comment by dberenbaum** [comment]
Have you taken a look at https://iterative.ai/blog/mlem-cv-model-deployment? It's not yolov8 or tensorflow, but it shows an example of deploying a CV model using MLEM.
